src/components/Points/index.js

```diff
@@ -53,9 +59,24 @@ const Points = ({
   pointWidthAccessor,
   strokeWidth,
 }) => {
-  const getX = x => boundedSeries(xScale(x));
-  const getY = y => boundedSeries(yScale(y));
-  const points = data.map(d => {
+  if (!drawPoints) {
+    // We should never wind up in this situation, but just in case...
+    return null;
```

Comment:
If `drawPoints` is `false`, it seems instead of `default rendering`, null will be rendered

Reply:
Which it should, right? `false` should result in no points being rendered.

Reply:
Oh nevermind -- I see the issue.